### PR TITLE
Create Jakarta EE 10 release landing page

### LINKED ISSUE #1463: Create the Jakarta EE 10 release landing page
Here is the timeline for this project: https://docs.google.com/spreadsheets/d/1Sx-KRJjS4tX-5Sz3kKnRUjrI5pdi0WcAsZeVb2Xx8vY/edit#gid=138610214

Please note that we are targeting May 31 for the release date as of now but it may change. 

Here is the link to the banner's design: https://xd.adobe.com/view/94e225c1-7027-48e5-8893-bcd8883725b6-c1b1/

Here is the link to the content doc for this page: https://docs.google.com/document/d/1ghGN8ApNU2nwLHVKqMvB3D_s8s05lpFMEV3ddiOkkxA/edit

Here is the link to the Jakarta EE 10 specifications image: https://xd.adobe.com/view/e2028f66-ab4d-4e08-82b0-bc91a3bca1ad-b751/ We need to add a title on this section above the image: Jakarta EE 10 Specifications 

Please note that for this release we are switching the Specification image section with the benefits and features section. so below the banner should be the features and benefits section. 

@ericpoirier please let me know if we should set up a quick call to go through the details before you start implementing.

### PR DESCRIPTION
**Pull request by ericpoirier**
Fixes #1463 

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by meKokabi** [comment]
Please find attached the required visual assets.
[VisualAssets.zip](https://github.com/jakartaee/jakarta.ee/files/8557128/VisualAssets.zip)

File: content/release/10/_index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+headline: "<img class='header-release-10-logo' src='images/jakarta10-logo.svg' alt='Jakarta EE 10'>"
+logo: "images/jakarta/jakarta_ee_logo_schooner_color_horizontal_default.png"
+links: [[href: "/compatibility/", text: "Download Compatible Products"], [href: "/specifications/", text: "Specifications"]]
+hide_page_title: true
+header_wrapper_class: "header-release-10"
+container: "container-fluid"
+footer_class: "footer-release-9"
+type: "release/10"
+hide_call_for_action: true
+press_release_link: "/news/jakarta-ee-9-1-released/"
```
Comment (Dexmaster) [inline content/release/10/_index.md]: Here should be press_release_link for Jakarta EE 10. (probably should be added when published, or provided link that will be used with anouncements)

File: content/release/10/_index.zh.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+headline: "<img class='header-release-10-logo' src='images/jakarta10-logo.svg' alt='Jakarta EE 10'>"
+logo: "images/jakarta/jakarta_ee_logo_schooner_color_horizontal_default.png"
+links: [[href: "/compatibility/", text: "下载兼容产品"], [href: "/specifications/", text: "规范"]]
+hide_page_title: true
+header_wrapper_class: "header-release-10"
+container: "container-fluid"
+footer_class: "footer-release-9"
+type: "release/10"
+hide_call_for_action: true
+press_release_link: "/news/jakarta-ee-9-1-released/"
```
Comment (Dexmaster) [inline content/release/10/_index.zh.md]: Here should be press_release_link for Jakarta EE 10

File: data/contributors/jakarta-ee-10.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  },
+  {
+    "login": "spericas",
+    "name": "Santiago Pericasgeertsen"
+  },
+  {
+    "login": "starksm64",
+    "name": "Scott M Stark"
+  }
+]
```
Comment (Dexmaster) [inline data/contributors/jakarta-ee-10.json:74]: Few more people will appear here after change of dates for generation.

File: js/contributor-cards/generate-contributor-lists.sh [hunk 4]
```diff
@@ -19,10 +19,10 @@
 # you need GH_TOKEN env variable for https://api.github.com/graphql
 CREDENTIALS=$GH_TOKEN:x-oauth-basic
 CURSOR=null
-LIST_FILE=data/contributors/${1:-jakarta-ee-9.1.json} # first argument ('jakarta-ee-9.1.json' by default)
+LIST_FILE=data/contributors/${1:-jakarta-ee-10.json} # first argument ('jakarta-ee-9.1.json' by default)
 EXCLUDED_LIST=data/contributors/excluded-list.json
-FROM_DATE=${2:-2020-11-21} # second argument ('2020-11-21' by default)
-TO_DATE=${3:-2021-05-18} # third argument ('2021-05-18' by default)
+FROM_DATE=${2:-2021-11-21} # second argument ('2021-11-21' by default)
```
Comment (Dexmaster) [inline js/contributor-cards/generate-contributor-lists.sh]: There is a typo here should be previous release date 2021-05-18

File: js/contributor-cards/generate-contributor-lists.sh [hunk 5]
```diff
@@ -19,10 +19,10 @@
...
+TO_DATE=${3:-2022-05-18} # third argument ('2022-05-18' by default)
```
Comment (Dexmaster) [inline js/contributor-cards/generate-contributor-lists.sh:25]: Could be this date, but basically generally planned Jakarta EE 10 release date here.

File: layouts/shortcodes/pages/release/10.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+      <div class="col-md-8 col-xs-12 vertical-align">
+        <div class="text-center">
+            <h3 class="font-100 text-secondary fw-400 font-exo"><em>129</em></h3>
+            <h4 class="text-blue fw-600 uppercase">{{ i18n "jakarta-ee-version-9_1-participation-repositories" }}</h4>
+            <hr class="jakarta-ee-hr" />
+        </div>
+    </div>
+
+      <div class="col-md-8 col-xs-12 col-xs-offset-6 col-md-offset-0 vertical-align">
```
Comment (Dexmaster) [inline layouts/shortcodes/pages/release/10.html]: Offsets break on smaller screens/mobile

**Review by Dexmaster — COMMENTED** [review]
Few small issues

File: layouts/shortcodes/pages/release/10.html [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+
+      <div class="col-md-8 col-xs-12 vertical-align">
+        <div class="text-center">
+            <h3 class="font-100 text-secondary fw-400 font-exo"><em>129</em></h3>
+            <h4 class="text-blue fw-600 uppercase">{{ i18n "jakarta-ee-version-9_1-participation-repositories" }}</h4>
+            <hr class="jakarta-ee-hr" />
+        </div>
+    </div>
+
+    <div class="col-md-8 col-xs-12 col-sm-offset-6 col-md-offset-0 vertical-align">
```
Comment (Dexmaster) [inline layouts/shortcodes/pages/release/10.html]: image
Issue still present

Comment (Dexmaster) [reply layouts/shortcodes/pages/release/10.html]: If you remove col-offsets (col-sm-offset-6) issue is fixed and it works fine for all sizes.

Comment (ericpoirier) [reply layouts/shortcodes/pages/release/10.html]: I updated the PR to remove the offset classes, which as you said, are not needed.

**Pull request merged**